test/FunctionalTests/Tests/DataOData/Tests/OData.Query.TDD.Tests/Semantic/Functional/PathFunctionalTests.cs

```diff
@@ -570,9 +570,8 @@ public void NavigationPropertyWithMissingEntitySetShouldThrow()
             var model = ModelBuildingHelpers.GetModelWithNavPropWithNoTargetSet();

```

Comment:
Also change the method name `NavigationPropertyWithMissingEntitySetShouldThrow`.